_includes/checklist.md

```diff
@@ -1,41 +1,91 @@
-* **Main repository:** <url>
-* **Running libraries:**
-    * <url>
-    * ...
-* **Site:** <url>
-* **Product manager:** @<username>
-* **Technical point of contact:** @<username>
+* **Main repository:** [url]
+* **Site:** [url]
+* **Product manager:** @[username]
+* **System Owner:** @[username of technical point of contact who will be the project representative in the Sprint]
+* **Infrastructure Lead:** @[username]
+* **ATO folder:** [url]
+* **Sprint notes:** [url]
 
 ## TODOs
 
-### Project team
+### ATO Sprint prerequisites
 
-Many of these tasks can be done in parallel. This is just a suggestion of priority.
+#### Infrastructure Lead
 
+- [ ] Set up an ATO intro meeting with the team.
+- [ ] Determine the [impact level](https://pages.18f.gov/before-you-ship/ato/levels/).
+- [ ] Add this issue to the `Backlog` of [the ATO Kanban board](https://github.com/18F/Infrastructure/projects/1).
+- [ ] Assign the appropriate labels to this issue.
+- [ ] Set up the project ATO folder.
+    - [ ] Create as a sub-folder of [the `ATOs` folder in Google Drive](https://drive.google.com/a/gsa.gov/folderview?id=0BynIxtx-CfkdckljM3BPSkdQT1U&usp=sharing) under 18F/OPP/PIF then "Work in progress". Link to it above.
+    - [ ] Add Rules of Engagement (RoE) template
+    - [ ] Add [System Security Plan (SSP)](https://pages.18f.gov/before-you-ship/ato/ssp/) template
+    - [ ] Add Project Plan template
+- [ ] Make a copy of the [notes doc template](https://docs.google.com/document/d/1EdcNyE1kkQve3tHyiV1QIRWNOBlTeh33lAbX0h4h18M/edit) in the [Sprinting Team folder](https://drive.google.com/open?id=1EdcNyE1kkQve3tHyiV1QIRWNOBlTeh33lAbX0h4h18M).
+    - [ ] Fill out the placeholders.
+    - [ ] Add link at the top of this issue.
+
+#### Project team
+
+##### Technical
+
+- [ ] Enable [protected branches](https://help.github.com/articles/about-protected-branches/) for the project repositor(ies).
+    * Get admin access via #admins-github, if needed.
+- [ ] Ensure that a staging environment is fully set up.
+    * This should be as [production](https://pages.18f.gov/before-you-ship/infrastructure/)-like as possible.
 - [ ] [Set up monitoring](https://pages.18f.gov/before-you-ship/infrastructure/monitoring/)
```

Comment:
No need for proof for these things. The link to the staging/testing environment will be at the top of the list already.